content/articles/index.md

```diff
@@ -0,0 +1,174 @@
+---
+layout: engineering-education
+status: publish
+published: true
+url: /Automatic-type-promotion/
+title: Automatic type promotion
+description: This article will cover a brief introduction to auto type promotion and method overloading and their relationships, reasons or circumstances under which you may be required to apply automatic type promotion and the different types or ways of implementing type promotion. 
+author: 
```

Comment:
Hey, @root5086  👋 .
Kindly link your author bio. 
I can't seem to find it.

Regards.

Reply:
Any update?